template.yaml

```diff
@@ -379,6 +382,7 @@ Resources:
           TABLE_NAME: !Ref PublicationsTableName
           BY_PUBLISHER_INDEX_NAME: !Ref PublicationsByOwnerIndexName
           BY_PUBLISHED_PUBLICATIONS_INDEX_NAME: !Ref PublishedPublicationsIndexName
+          AWS_REGION: ${AWS::REGION}
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Found an embedded parameter outside of an &#34;Fn::Sub&#34; at Resources/NvaPublicationsByOwnerFunction/Properties/Environment/Variables/AWS_REGION